frontend/app/updateRollButton.js

```diff
@@ -27,7 +32,7 @@ export function requestCompareRolls(JSONSend = sendJSON.sendJSON) {
     const details = getCookie.checkUserDetails();
     const id = details.user_id;
     JSONSend({
-        serverAddress: 'cgi-bin/compare_dice_rolls.py',
+        serverAddress: 'cgi-bin/roll_die.py',
```

Comment:
According to pages.py, the serverAddress should be 'cgi-bin/roll_dice.py' :smile: .